.github/workflows/ci.yaml

```diff
@@ -29,8 +29,25 @@ on:
       - ".gitmodules"
       - ".gitattributes"
       - ".editorconfig"
-
 jobs:
+  lint:
+    name: Check code linting
+    runs-on: ubuntu-latest
+    env:
+      PIPENV_DEFAULT_PYTHON_VERSION: ${{ matrix.python-version }}
+      PYTHONWARNINGS: ignore:DEPRECATION
+      PYTHONIOENCODING: "utf-8"
+      GIT_ASK_YESNO: "false"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - run: |
+          python -m pip install --upgrade pip setuptools wheel --upgrade-strategy=eager
+          python -m pip install -e . --upgrade
```

Comment:
Probably don't have to install pipenv to lint it.

Reply:
Yes, maybe it's enough to just install pre-commit...